File: src/main/java/com/kerosenelabs/billtracker/model/RecurringType.java
package com.kerosenelabs.billtracker.model;

/**
 * Represents the Recurring Types of expenses or income. For example,
 * your mortgage might be billed on the first of every month, but you
 * might get paid every two weeks.
 */
public enum RecurringType {
    FIXED_DATE,
    FIXED_INTERVAL
}

File: src/main/java/com/kerosenelabs/billtracker/model/RecurringType.kt
package com.kerosenelabs.billtracker.model

/**
 * Represents the Recurring Types of expenses or income. For example,
 * your mortgage might be billed on the first of every month, but you
 * might get paid every two weeks.
 */
enum class RecurringType {
    DATE,
    INTERVAL
}
